### DISCUSSION TITLE
[ FF7 ] Crash on game launch from 7th Heaven (2.0)

### DISCUSSION THREAD

**Discussion by kandrew152** [julianxhokaxhiu/FFNx#170] [Q&A]
I'm getting an "Oops, something bad happened" crash. The game window opens but is still black when it happens. Setup attached:

Fresh and working install of FF7 Steam + 7th Heaven (2.0). Followed OatBran's guide for install, converted Qhimm packages to DDS files, and have been playing since yesterday. Woke computer up earlier and couldn't get it to start again.

Links:
FFNx.log: https://drive.google.com/file/d/1Gne-Uh0Lc_0H_u_sojtPMhM4FKwSURKZ/view?usp=sharing
Crash.ff7: https://drive.google.com/file/d/1ZqRxTfwTUqtCPXxdE7T0Oy9wb61-JG0D/view?usp=sharing

**Comment by julianxhokaxhiu** [comment]
Hi @kandrew152 

Thanks for having me approached. I would like to download your log file but it's protected. Feel free to attach it here on Github instead both files ( just zip them with no password ).

Related to the setup, if you use 7th 2.x you need to download the 1998 release, not the Steam one ( even if you use the Steam setup, 7th uses the 1998 exe to start the game, therefore you need the 1998 ZIP ).

Still it would be worth to check the log files anyway.

Looking forward.

**Comment by kandrew152** [comment]
[FFNx Logs.zip](https://github.com/julianxhokaxhiu/FFNx/files/4682458/FFNx.Logs.zip)

Thanks, that was much easier. I did install using the 1998 release executable, so that shouldn't be the issue.

It appears that it was having a problem with one of the mods I had enabled. I'm narrowing down which one but after disabling them all I was able to get it to run again. Hopefully the log files will tell you?

**Comment by julianxhokaxhiu** [comment]
@kandrew152 than you for the log file. It appears some of the mods you are using are not providing some game required files. This is why you get a crash. Although this is not related to FFNx itself ( and you can easily verify that by running the vanilla game or turning off some mods until you find which one it is ).

For better mods discussion and to get which one is the culprit and how to fix it, I suggest you joining the Discord channel in the README and ask in the #support channel :)

Thank you and see you there!
